contracts/dollar/package.json

```diff
@@ -13,18 +13,18 @@
     "url": "https://github.com/ubiquity/uad-contracts.git"
   },
   "scripts": {
-    "clean": "yarn hardhat clean",
```

Comment:
Was this a merge issue thing? This file shouldn't have been modified.

Reply:
I think this is technically correct, to exclude `yarn`

If it works then it settles the matter.